doc/collections/_adrs/0001-decision-to-record-decisions.md

```diff
@@ -0,0 +1,17 @@
+----
+technical_design:
+----
+
+
+
+| ## ADR 0001: Decision to Record Decisions | &nbsp; | &nbsp; | &nbsp; |
+| --- | --- | --- | ---- |
+| Date | 2023-04-19 | |
+| Authors | Bard | @DerekNonGeneric | @DynamicWebPaige |
+| Status | Approved | |
+| Decision | See [abstract](./#abstract)&hellip; | |
+
```

Comment:
```suggestion
| <strong>ADR 0001: Decision to Record Decisions</strong> | &nbsp; | &nbsp; | &nbsp; |
| ---:|:---:|:---:|:----:|
| Date | 2023-04-19 | |
| Authors | Bard | @DerekNonGeneric | @DynamicWebPaige |
| Status | Approved | |
| Decision | See [abstract](./#abstract)&hellip; | |
```

![Screenshot 2023-12-27 at 17-23-52 meta-feature add support for CentOS 7 · Issue #16 · OpenINF_docker-fisher](https://github.com/OpenINF/.github/assets/17770407/46d1dfda-4aa3-431e-9071-3e1a0db3f9f6)

Reply:
ok, the AIP purpose and guidelines are pretty, had me think of going tabular:

- https://github.com/aep-dev/aep.dev/tree/main/aep/general

Reply:
State Approved as opposed to Unapproved, but likely other states exist.

Reply:
**States**

![image](https://github.com/OpenINF/.github/assets/17770407/5402a12f-4558-45d5-a817-d6d5433f8ee7)